table/options/options.go

```diff
@@ -112,8 +114,21 @@ func (settings timeToLiveSettings) ApplyCreateTableOption(d *CreateTableDesc, a
 	d.TtlSettings = (*TimeToLiveSettings)(&settings).ToYDB()
 }
 
-func WithTimeToLiveSettings(settings TimeToLiveSettings) CreateTableOption {
-	return timeToLiveSettings(settings)
+// WithTTL defines TTL settings in CreateTable request
+func WithTTL(columnName string, ttl time.Duration, opts ...timeToLiveSettingsOption) CreateTableOption {
```

Comment:
I suggest add function WithTimeToLiveSettings only - for uniformity.
Other functions no double variants, it receive settings object only.